testsuite/forge/src/interface/prometheus_metrics.rs

```diff
@@ -108,6 +108,7 @@ pub async fn fetch_system_metrics(
 pub enum LatencyBreakdownSlice {
     QsBatchToPos,
     QsPosToProposal,
```

Comment:
remove these two, and then you'll see what else you'll need to modify

this enum is not serialized, so we are free to remove elements from here

Reply:
Sounds good 👍